examples/metrics/plot_itakura.py

```diff
@@ -0,0 +1,114 @@
+"""
+=====================
+Itakura parallelogram
+=====================
+
+This example shows how to set the parameters of the Itakura parallelogram when
+computing the Dynamic Time Warping (DTW) between two time series.
+It is implemented in :func:`pyts.metrics.dtw.itakura_parallelogram`.
+"""
+
+# Author: Johann Faouzi <johann.faouzi@gmail.com>
+#         Hicham Janati <hicham.janati@inria.fr>
+# License: BSD-3 Clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+from pyts.metrics import itakura_parallelogram
+from pyts.metrics.dtw import _get_itakura_slopes
+
+# We write a function to visualize the sakoe-chiba band for different
```

Comment:
I would advocate for a longer description at the beginning of the file, explaining the different things that we want to visualize, and one single plot with several rows.